modules/apps/store/extensions/assets/webapp/pages/asset.jag

```diff
@@ -120,11 +120,26 @@ require('/modules/store.js').exec(function (ctx) {
 
         var defaultApplicationName = "DefaultApplication";
 
-        var enterpriseSubscription = subscriptionService.getSubscription(appInfo, defaultApplicationName, "ENTERPRISE", authenticator.getLoggedInUser().username).subscription; 
+        var enterpriseSubscription;
```

Comment:
In code line 124-139, we have duplicated same if condition twice here. Also the logic need to re-checked.